### PR TITLE
Add GRIM The Curse

### PR DESCRIPTION
**Pull request by Superfly-Johnson**
That's it really.

### CODE REVIEW

File: masterlist.yaml [hunk 1]
```diff
@@ -5798,3 +5798,12 @@ plugins:
...
       - 'Farm_DLCworkshop03.esp'
+
+  - name: 'GRIM.esp'
+    url: [ 'https://www.nexusmods.com/fallout4/mods/46711' ]
+    dirty:
+      - <<: *quickClean
+        crc: 0x613CC196
+        util: '[FO4Edit v4.0.3](https://www.nexusmods.com/fallout4/mods/2737)'
+        itm: 227
+        udr: 2
```
Comment (MacSplody) [inline masterlist.yaml]: There needs to be a newline at the end of file.
```suggestion
        udr: 2

```

**Review by MacSplody — CHANGES_REQUESTED** [review]
The rest looks fine, just add a newline at the end of file.

**Comment by Superfly-Johnson** [comment]
Should be fixed. I think? I can see the added newline in my text editor but not on the github file viewer.

File: masterlist.yaml (hunk 1 above)
Comment (Superfly-Johnson) [reply masterlist.yaml]: Done and squashed. Somehow I can only see the change in my text editor, sorry if it actually didn't work.

**Review by MacSplody — APPROVED** [review]

**Comment by MacSplody** [comment]
@Superfly-Johnson Thanks.

**Pull request merged**